uiowa/uiowa
# Feature: Featured image backgrounds and sidebar menus for pages

Resolves #1484. 
Resolves #1617.
# Updates
* Changed base profile config for page entity view to use layout builder for managing layout.
* Changed base profile config for article entity view to use layout builder for managing layout.
* Add a layout template for page content with sidebar.
* Add a layout template for content over a background image.
* Add styles to accommodate new layouts.
* Update article and page template to remove featured image, breadcrumb, and title.
* Adds Layout Builder Direct Add module to base profile.

# How to test
Ideally, this should be checked against several sites. We want to make sure it doesn't break anything for uiowa.edu, while also seeing that the functionality is updated for brand.uiowa.edu, hawkeyemarchingband.uiowa.edu, and any other sites you care to check against.

- `blt frontend` 
- `blt dsa` for any sites you are checking.
- Check a page that does not have a featured image. Make sure that the title and breadcrumb show as normal.
- Add a featured image to the same page and save. Confirm that it now show the featured image with the breadcrumb and title on top of it and that the regular breadcrumb and title don't appear.
- Check an article that does not have a featured image for the same as above.
- Repeat adding a featured image to the article and confirm the same as above.
- Attach a page to a parent menu item, verify that sidebar menu appears on left-hand sidebar. 
- Set "hide page title" in Page Options and verify that page title is hidden when not using a featured image. 
- Set "Remove sidebar regions" in Page Options on a page that is attached to a parent item in the main menu.
